src/main/kotlin/net/minestom/dependencies/DependencyGetter.kt

```diff
@@ -20,7 +20,7 @@ class DependencyGetter {
     fun addMavenResolver(repositories: List<MavenRepository>) = addResolver(MavenResolver(repositories))
 
     fun get(id: String, targetFolder: File): ResolvedDependency {
-        for(resolver in resolverList) {
```

Comment:
Replaced legacy for-each with lambda forEach (It is common to use lambdas instead of legacy for-eachs)